.github/workflows/release.yml

```
@@ -113,6 +113,9 @@ jobs:
           path: artifacts
       - name: Flatten artifacts
         run: mkdir -p release && find artifacts -type f -exec cp {} release/ \;
+      - name: Generate SHA256SUMS
+        working-directory: release
+        run: sha256sum * > SHA256SUMS
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:1:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)